src/coreComponents/physicsSolvers/inducedSeismicity/ExplicitQDRateAndState.cpp

```diff
@@ -184,31 +103,29 @@ real64 QuasiDynamicEQRK32::solverStep( real64 const & time_n,
 
     stepRateStateODEAndComputeError( dtAdaptive, domain );
     // Update timestep based on the time step error
-    real64 const dtNext = setNextDt( dtAdaptive, domain );
-    if( m_successfulStep ) // set in setNextDt
+    evalTimestep( domain );
```

Comment:
a bit confused about the time stepping logic here
can it be framed into standard GEOS sub time step loop or there is something fundamentally different here?

Reply:
To clarify what is going on we use adaptive explicit time stepping of the rate-and-state ODE:s for slip and state. The adaptive time step update is based on the estimated error in these two fields. The call to evalTimestep checks whether the time step was successful (within error tolerances) and computes an update factor for changing the time step. If the time step was unsuccessful we retry it with the updated time step. If it succeeded, the call to setNextDt after we've exited solverStep will update the time step for the next cycle.

Someone more familiar with GEOS perhaps can answer whether this would fit into the sub time step loop.